tests/testthat/test-search.R

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+context('test bcdc_search')
+
+
+test_that('works with a record that has no resource',{
+  expect_silent(bcdc_search("Major Railways"))
```

Comment:
Does this fail without this change? Not sure it would, as it's just in the print method isn't it?

Reply:
Oh i see. This doesn't trigger the print method? What if I did added another set of parentheses around the `bcdc_search` call?

Reply:
Or what about this `expect_is(bcdc_search("Major Railways"), "bcdc_recordlist")`?

Reply:
No I think that would still pass before this fix. We might not actually be able to test it?

Reply:
This seems to work: 

```r
test_that('works with a record that has no resource', {
  output_path <- tempfile()
  suppressWarnings(
    verify_output(output_path, {
      bcdc_search("Major Railways")
    })
  )
  expect_false(any(grepl("Error", readLines(output_path))))
  expect_true(any(grepl("no resources", readLines(output_path))))
})
```

I tried it on `master` and it failed, on `search-print` it passes.

Reply:
Great idea! I've committed this with you as the author. Will merge if tests (mostly) pass 😉